### PR TITLE
feat: allow hiding current screen

### PR DESCRIPTION
**Pull request by PoolloverNathan**

### CODE REVIEW

**Review by TheBunnyMan123 — CHANGES_REQUESTED** [review]

File: common/src/main/resources/goofyplugin-common.mixins.json
[the remark this answers is not in the document]
Comment (PoolloverNathan) [reply common/src/main/resources/goofyplugin-common.mixins.json:1]: the plugin makes it alphabetical automatically

File: wiki/Misc.md [hunk 1]
```diff
@@ -5,7 +5,7 @@
 - `goofy:stopAvatar()` stops the avatar as if an error occured 
 - `goofy:loadLocalAvatar(path)` loads a local avatar 
 - `goofy:uploadAvatar()` uploads the current avatar
-
+git switch -c hide-gui
```
[the remark this answers is not in the document]
Comment (PoolloverNathan) [reply wiki/Misc.md:8]: oops fuck

**Comment by TheBunnyMan123** [comment]
did you test this?

**Comment by PoolloverNathan** [comment]
> did you test this?

yes, and it works

**Comment by PoolloverNathan** [comment]
the mixin I added feels sketchy, but it's what Fabric API does, so it should be fine

**Review by TheBunnyMan123 — APPROVED** [review]
LGTM

**Pull request merged**